lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
+        lti_url=None
```

Comment:
@robrap As far as I can tell, the lti_url provided here would be what someone would use to add the block as an LTI provider to an LTI consumer.

The other endpoint you reference comes into play when edx is acting as the LTI consumer. When a student interacts with an edX LTI consumer component which has been added to a course, that LTI consumer component will include a URL in the LTI launch request sent to the LTI provider which the provider can use to pass grades back to the edx lms. The referenced endpoint above looks like it is intended to allow someone to get a list of all such URLs for a given course. Happy to step through it in person.